infrastructure/sprod.tfvars

```diff
@@ -1,50 +1,44 @@
+//module vars
+asp_name = "probate-sprod"
+asp_rg = "probate-sprod"
 env = "sprod"
+external_host_name = "probate-caveats.sprod.platform.hmcts.net"
+
+//app settings vars
 deployment_env = "test"
 
-vault_section = "test"
+feature_toggles_api_url = "http://rpe-feature-toggle-api-sprod.service.core-compute-sprod.internal"
+
+idam_service_api = "http://rpe-service-auth-provider-sprod.service.core-compute-sprod.internal"
+idam_user_host = "http://betaDevBccidamAppLB.reform.hmcts.net"
 
 packages_environment = "test"
 packages_version = "3.0.0"
 
-probate_frontend_hostname = "probate-frontend-sandbox-sprod.service.core-compute-sprod.internal"
-external_host_name = "probate.sprod.platform.hmcts.net"
-outbound_proxy = ""
-
-probate_google_track_id = "UA-93598808-1"
-
-probate_redis_url = "betaDevBprobatecache01.reform.hmcts.net"
-
-f5_redis_listen_port = "6379"
-
-probate_frontend_https = "true"
-probate_frontend_use_auth = "false"
-
-payment_return_url = "https://probate-frontend-sandbox-sprod.service.core-compute-sprod.internal/payment-status"
-
 payment_create_url = "https://test.payments.reform.hmcts.net:4421/users/userId/payments"
 
-idam_service_api = "http://betadevbccidams2slb.reform.hmcts.net"
-
-probate_private_beta_auth_url = "https://idam-test.dev.ccidam.reform.hmcts.net/login"
-
-idam_user_host = "http://betaDevBccidamAppLB.reform.hmcts.net"
-
 probate_deployment_env = "test"
 
-
+probate_frontend_https = "true"
 probate_frontend_port = "3101"
-
+probate_frontend_use_auth = "false"
 probate_frontend_use_idam = "true"
-
 probate_frontend_use_redis = "true"
 
-reform_envirionment_for_test = "saat"
+probate_google_track_id = "UA-93598808-1"
 
-probate_business_service_url = "http://betaDevBprobateApp01.reform.hmcts.net:4101/validate"
+probate_private_beta_auth_url = "https://idam-test.dev.ccidam.reform.hmcts.net/login"
 
+probate_business_service_url = "http://betaDevBprobateApp01.reform.hmcts.net:4101/validate"
```

Comment:
probate_business_service_url = "http://betaDevBprobateApp01.reform.hmcts.net:4101/validate"
probate_persistence_service_url = "http://betaDevBprobateApp01.reform.hmcts.net:4103/formdata"
probate_submit_service_url = "http://betaDevBprobateApp01.reform.hmcts.net:4102/submit"

Reply:
remove this

Reply:
removed